app/assets/stylesheets/base.scss

```diff
@@ -21,3 +21,4 @@
 @import "modules/caret";
 @import "modules/search_across";
 @import "modules/exhibit_search_typeahead";
+@import "modules/browse";
```

Comment:
`@import` is deprecated in SASS.  Do we want to continue to use this? https://sass-lang.com/blog/import-is-deprecated/

Reply:
I think we need to get to `dart-sass` before we can stop using `@import`. We have an issue for it https://github.com/sul-dlss/exhibits/issues/2445